asciigraf/asciigraf.py

```diff
@@ -274,7 +334,7 @@ def draw(edge_chars, nodes=None):
             cursor = Point(0, position.y)
         if cursor.x < position.x:
             string += ' ' * (position.x - cursor.x)
-            cursor = Point(*position)
+            cursor = Point(position.x, position.y)
```

Comment:
You could also solve this by fleshing out the `Point` class with a `copy` method. But actually, I think my current mood is that it'd be nicer if Point was immutable. This is all just style fussiness though.

Reply:
I like the idea of Point being immutable, then we can just do `cursor = position`